chef/cookbooks/crowbar/recipes/default.rb

```diff
@@ -242,34 +242,6 @@
     "/var/cache/chef/solr/data"
   end
 
-  # If all nodes are done installing, change the machine-install password so it
-  # can't be reused. This will change it on every chef run.  This only changes
-  # the install key on the admin node. This will be injected into future nodes
-  # that are added later.
-  all_nodes = node_search_with_cache("*:*")
-  unready_nodes = node_search_with_cache("NOT state:ready")
-  nodes_ready = all_nodes.length > 1 && unready_nodes.empty?
-  live_nodes = all_nodes.length
-  # Change the install key when all nodes are ready and the number of nodes in
-  # the deployment has changed since the last time it was set
-  if nodes_ready && live_nodes != node["crowbar"]["known_nodes"]
-    node["crowbar"]["known_nodes"] = live_nodes
-    secret = SecureRandom.hex(64)
-    node["crowbar"]["users"]["machine-install"]["password"] = secret
-    file "/etc/crowbar.install.key" do
-      content "machine-install:#{secret}\n"
-      owner "root"
-      group "root"
-      mode "0o600"
-    end
-    execute "update crowbar proposal with new machine-install password" do
-      command "crowbarctl proposal edit crowbar default \
-               --data \"$(crowbarctl  proposal show crowbar default --json | \
-               /opt/dell/bin/json-edit \
-                 --attribute attributes.crowbar.users.machine-install.password \
-                 --value #{secret} - )\""
-    end
-  end
```

Comment:
This commit could be squashed into 004169812f8b2a49354710b080e9a431d1a444ef, no need to add this and then undo it in the same PR.

Reply:
Yeah, I thought we'd merge your PR first and then consider only the new commits. Let me clean this up further, then.